### ISSUE TITLE
No template resource can be found when generating documentation with docfx.csproj

### ISSUE THREAD

**Issue by Kavignon** [dotnet/docfx#4036]
**DocFX Version Used**: 
2.39.2 to 2.40.10
**Template used**: (`default` or `statictoc` or contain custom template)
both default and statictoc 
**Steps to Reproduce**:

1.  Adding a class library to a .NET solution
2.  Generate documentation from All.sln using 'DocFX' as startup project + providing 'docfx.json' from referring to the class library
3. Run the application

**Expected Behavior**:
Having both the intermediate files along with the generated HTML files.
**Actual Behavior**:
![image](https://user-images.githubusercontent.com/6740474/52745784-f4895780-2fad-11e9-8a0c-d6107b7da8e7.png)
![image](https://user-images.githubusercontent.com/6740474/52745798-feab5600-2fad-11e9-95e2-1a607611987d.png)

When using the docfx.exe from the release notes or the NuGet package, the tool is able to generate the documentation properly. 

Although for some that may be ok, I have some update to do the code and I can't validate those changes when the docFX.csproj is blocking me from generating documentation.

**Comment by superyyrrzz** [comment]
Does your repo contain folder named `default`?

**Comment by Kavignon** [comment]
@superyyrrzz No, but that's not the issue. The templates files from the template folder were set to ignore in the git ignore. 
When pulling the code from the repository or from the release notes, I wasn't able to generate the documentation of from that.

It was only with either the NuGet or the file exe that came with the docfx.zip from the release notes that worked. From there, I used DotPeek to analyze the difference between the two EXE and I found that that the only embedded resource I had from the code was the langword.yml when the EXE had some much more. 

That was my issue and I fixed it by simply moving over those files in my fork.
